src/main/java/jenkins/plugins/extracolumns/CronTriggerColumn.java

```diff
@@ -0,0 +1,122 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2012, Frederic Gurr
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package jenkins.plugins.extracolumns;
+
+import java.util.Map;
+import java.util.Calendar;
+import java.text.DateFormat;
+
+import org.kohsuke.stapler.DataBoundConstructor;
+
+import hudson.Extension;
+import hudson.model.Job;
+import hudson.model.AbstractProject;
+
+import hudson.views.ListViewColumnDescriptor;
+import hudson.views.ListViewColumn;
+
+import hudson.triggers.Trigger;
+import hudson.triggers.TimerTrigger;
+import hudson.triggers.TriggerDescriptor;
+
+import hudson.scheduler.CronTabList;
+import hudson.scheduler.Hash;
+
+public class CronTriggerColumn extends ListViewColumn {
+
+    @DataBoundConstructor
+    public CronTriggerColumn() {
+        super();
+    }
+
+
+    public String getCronTrigger(@SuppressWarnings("rawtypes") Job job) {
+	String cronTrigger = "";
```

Comment:
Instead of an empty string, it should return "N/A" to be in line with other columns, e.g. Test Results.